A production build with Vite 3 beta fails because of the service worker configuration

### Description of the bug
Running a starter project in production using `mvn -Pproduction` with Vite 3 fails with
```
[ERROR] [vaadin:inject-manifest-to-sw] Unable to find a place to inject the manifest. This is likely because swSrc and swDest are configured to the same file. Please ensure that your swSrc file contains the following: self.__WB_MANIFEST
[ERROR] error during build:
[ERROR] Error: Unable to find a place to inject the manifest. This is likely because swSrc and swDest are configured to the same file. Please ensure that your swSrc file contains the following: self.__WB_MANIFEST
[ERROR]     at injectManifest (/.../node_modules/workbox-build/build/inject-manifest.js:144:19)
```

### Minimal reproducible example
```
vaadin init test-vite3
cd test-vite3
npm i vite@3.0.0-beta.0
echo "com.vaadin.experimental.viteForFrontendBuild=true" > src/main/resources/vaadin-featureflags.properties
mvn -Pproduction                                                                                            
```

### Expected behavior
It runs

### Versions:
    - Vaadin / Flow version: 23.1.1
    - Java version: 17.0.1
    - OS version: macOS

--- a/flow-server/src/main/resources/vite.generated.ts
+++ b/flow-server/src/main/resources/vite.generated.ts
@@ -410,6 +410,10 @@ export const vaadinConfig: UserConfigFn = (env) => {
         '@vaadin/router',
         '@vaadin/vaadin-license-checker',
         '@vaadin/vaadin-usage-statistics',
+        'workbox-core',
+        'workbox-precaching',
+        'workbox-routing',
+        'workbox-strategies'
       ]
     },
     plugins: [

